integrations/whatsapp/src/conversation.ts

```diff
@@ -1,63 +1,79 @@
-import { RuntimeError } from '@botpress/client'
+import { Conversation, RuntimeError } from '@botpress/client'
+import { IntegrationContext } from '@botpress/sdk'
+import {
+  PhoneNumberIdTag,
+  UserPhoneTag,
+  TemplateNameTag,
+  name,
+  TemplateLanguageTag,
+  TemplateVariablesTag,
+} from 'integration.definition'
 import { WhatsAppAPI, Types } from 'whatsapp-api-js'
 import z from 'zod'
 import { IntegrationLogger } from '.'
 import * as botpress from '.botpress'
+import { Channels } from '.botpress/implementation/channels'
```

Comment:
**nit:** you could use `botpress.channels.Channels` or `botpress.channels.channel.Channel` so we know where it comes from without looking at the imports \+ no need to dupplicate the `.botpress` import

Reply:
OK, maybe you can give me a quick walkthrough of how to easily/consistently find all these common types in the `.botpress` build folder, I feel sometimes it's a bit weird/convoluted trying to find them.